test/e2e/allocation.cancel.test.js

```diff
@@ -0,0 +1,22 @@
+import createAllocation from './_allocation'
+import { pmuUser } from './_roles'
+import { allocations, newAllocation } from './_routes'
+import { allocationCancelPage, page } from './pages/'
+
+fixture('Cancel allocation').beforeEach(async t => {
+  await t.useRole(pmuUser).navigateTo(newAllocation)
+  await createAllocation()
```

Comment:
I think it might be nice to use a similar approach to the moves fixtures used with the edit tests to create an allocation fixture using the API service rather than needing to go through all the steps each time.

Reply:
Whilst I love the approach you mention, I am not entirely sure it makes sense here. I would make helpers that will be used literally only once, as there are only 2 pages in the create allocation. It would be nice, but does it represent business / technical value?

Reply:
I think the long term technical benefit would be efficiency and re-usability. I imagine we will need to create an allocation to test further parts of that section of the app? For example, testing the lists with allocations in different states and filters applied. It will be more efficient to not have to run through those pages each time. 

The business value comes from the codebase being easier to work on as we continue. If we do each ticket in isolation and never consider the wider codebase it becomes difficult to maintain or handover in the future. 

if there is a solution that is more efficient and can be achieved in a similar amount of time we should aim for that.